File: 1635-number-of-good-pairs/1635-number-of-good-pairs.java
class Solution {
    public int numIdenticalPairs(int[] nums) {
        int goodPairs = 0;
        for(int i=0;i<nums.length-1;i++){
            for(int j=i+1;j<nums.length;j++){
                if(i < j && nums[i] == nums[j]){
                    goodPairs++;
                }
            }
        }
        return goodPairs;
    }
}

File: 1635-number-of-good-pairs/1635-number-of-good-pairs.py
class Solution:
    def numIdenticalPairs(self, nums):
        count_map = {}
        good_pairs = 0

        for num in nums:
            good_pairs += count_map.get(num, 0)
            count_map[num] = count_map.get(num, 0) + 1

        return good_pairs
